zgw_consumers/settings.py

```diff
@@ -1,11 +1,17 @@
+import os
+
 from django.conf import settings
-from django.core.cache import DEFAULT_CACHE_ALIAS
 
 from .constants import NLXDirectories
 
 NLX_DIRECTORY_URLS = {
-    NLXDirectories.demo: "https://demo-directory-api.commonground.acc.utrecht.nl/",
-    NLXDirectories.prod: "https://directory-api.commonground.utrecht.nl/",
+    NLXDirectories.demo: os.environ.get(
+        "NLX_DIRECTORY_URL_DEMO",
+        "https://nlx-directory-ui.commonground.acc.utrecht.nl/",
+    ),
+    NLXDirectories.prod: os.environ.get(
+        "NLX_DIRECTORY_URL_PROD", "https://nlx-directory-ui.commonground.utrecht.nl/"
+    ),
```

Comment:
Since we're only reading and not writing envvars, let's use `getenv` instead

```suggestion
    NLXDirectories.demo: os.getenv(
        "NLX_DIRECTORY_URL_DEMO",
        "https://nlx-directory-ui.commonground.acc.utrecht.nl/",
    ),
    NLXDirectories.prod: os.getenv(
        "NLX_DIRECTORY_URL_PROD", "https://nlx-directory-ui.commonground.utrecht.nl/"
    ),
```